sharding-proxy/sharding-proxy-frontend/sharding-proxy-frontend-mysql/src/main/java/org/apache/shardingsphere/shardingproxy/frontend/mysql/command/query/binary/prepare/MySQLComStmtPrepareExecutor.java

```diff
@@ -59,11 +55,9 @@ public MySQLComStmtPrepareExecutor(final MySQLComStmtPreparePacket packet, final
         SQLStatement sqlStatement = logicSchema.getParseEngine().parse(packet.getSql(), true);
         int parametersCount = sqlStatement.getParametersCount();
         result.add(new MySQLComStmtPrepareOKPacket(++currentSequenceId, PREPARED_STATEMENT_REGISTRY.register(packet.getSql(), parametersCount), getNumColumns(), parametersCount, 0));
-        TablesContext tablesContext = new TablesContext(sqlStatement);
         for (int i = 0; i < parametersCount; i++) {
             // TODO add column name
-            result.add(new MySQLColumnDefinition41Packet(++currentSequenceId, schemaName,
-                    tablesContext.isSingleTable() ? tablesContext.getSingleTableName() : "", "", "", "", 100, MySQLColumnType.MYSQL_TYPE_VARCHAR, 0));
+            result.add(new MySQLColumnDefinition41Packet(++currentSequenceId, "", "", "", "?", "", 0, MySQLColumnType.MYSQL_TYPE_VAR_STRING, 0));
```

Comment:
We need to explain hard code with empty string

Reply:
OK. I'll add this in the reference issue.